arm-datalake-analytics/job/2016-03-20-preview/swagger/job.json

```diff
@@ -0,0 +1,1017 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DataLakeAnalyticsJobManagementClient",
+    "description": "Creates an Azure Data Lake Analytics job client.",
+    "version": "2016-03-20-preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "internalConstructors": true
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{accountName}.{adlaJobDnsSuffix}",
```

Comment:
I saw that, and we made the (painful) decision to take advantage of your change and move accountName back to the front, since it is much more consistent and in line with what we had in Hyak land. If it makes it more explicit, I can include the positionInOperation: first tag, if you like?